plugins/node/instrumentation-cucumber/src/instrumentation.ts

```diff
@@ -72,11 +69,8 @@ export class CucumberInstrumentation extends InstrumentationBase {
           });
           return moduleExports;
         },
-        (moduleExports, moduleVersion) => {
+        moduleExports => {
```

Comment:
There are a few instrumentations still that do not have the moduleVersion removed even though they are no longer being used. Is that intentional? If not, can we update those as well for consistency? Happy to help take on that extra bit if you need a hand, there's a lot here 😃  Also I realize we'll probably have some merge conflicts to deal with between this and #2091 depending on which one merges first.

Reply:
Right! thanks for spotting it. I removed the `moduleVersion` everywhere it is not needed anymore. Hope I didn't miss somewhere